libkeepass/kdb4.py

```diff
@@ -90,7 +90,7 @@ def write_to(self, stream):
         
         :arg stream: A writeable file-like object or IO buffer.
         """
-        if not (isinstance(stream, io.IOBase) or isinstance(stream, file)):
+        if not (isinstance(stream, io.IOBase)) or (not IS_PYTHON_3 and not isinstance(stream, file)):
```

Comment:
`IS_PYTHON_3` is not imported.
And wouldn't it make sense to make use of the `_is_file()` function instead?